app/scripts/migrations/082.test.js

```diff
@@ -1,6 +1,11 @@
 import { v4 } from 'uuid';
 import { migrate, version } from './082';
 
+const sentryCaptureExceptionMock = jest.fn();
+
+global.sentry = global.sentry || {};
```

Comment:
Is `global.sentry` ever set in this context?

Reply:
oh good question...

no, I don't think it is

Reply:
this redundancy has been removed in all places now